tests/test_footprint1.py

```diff
@@ -747,6 +747,20 @@ def testTransform(self):
         self.assertNotEqual(fpTarget.getArea(), fpTarget2.getArea())
         self.assertEqual(fpTarget.getArea(), fpTarget3.getArea())
 
+        # Test that peakCatalogs get Transformed correctly
+        truthList = [(x, y, 10) for x, y in zip(range(-2, 2), range(-1, 3))]
+        for value in truthList:
+            fpSource.addPeak(*value)
+        linTrans = afwGeom.LinearTransform(np.matrix([[2, 0],
+                                                      [0, 2]], dtype=float))
+        linTransFootprint = fpSource.transform(linTrans, fpSource.getBBox(),
+                                               False)
+        for peak, truth in zip(linTransFootprint.peaks, truthList):
+            # Multiplied by two because that is the linear transform scaling
+            # factor
+            self.assertEqual(peak.getIx(), truth[0]*2)
+            self.assertEqual(peak.getIy(), truth[1]*2)
```

Comment:
I suggest making the linear transform scaling factor a named constant (used when creating the linear transform above and again here).